cypress/support/cypress-support/src/main.md

```diff
@@ -52,89 +52,3 @@ request: cy.sendMessagetoApp('mac_appId_FCS',mac_appId_FCA,{"communicationMode":
 ```
 
 Note: Response can vary depending on the execution.
-
```

Comment:
These functions for the most part have just been moved to the plugins file right? Could we create a readme there and add this documentation instead of removing it all together?

Reply:
added the md file in plugins for testDataProcessor